backend/src/schema/resolvers/groups.spec.js

```diff
@@ -2982,4 +2967,171 @@ describe('in mode', () => {
       })
     })
   })
+
+  describe('RemoveUserFromGroup', () => {
+    beforeAll(async () => {
+      await seedComplexScenarioAndClearAuthentication()
+    })
+
+    afterAll(async () => {
+      await cleanDatabase()
+    })
+
+    describe('unauthenticated', () => {
+      it('throws an error', async () => {
+        await expect(
+          mutate({
+            mutation: removeUserFromGroupMutation(),
+            variables: {
+              groupId: 'hidden-group',
+              userId: 'usual-member-user',
+            },
+          }),
+        ).resolves.toMatchObject({
+          errors: expect.arrayContaining([
+            expect.objectContaining({
+              message: 'Not Authorized!',
+            }),
+          ]),
+        })
+      })
+    })
+
+    describe('authenticated', () => {
+      describe('as usual member', () => {
+        it('throws an error', async () => {
+          authenticatedUser = await usualMemberUser.toJson()
+          await expect(
+            mutate({
+              mutation: removeUserFromGroupMutation(),
+              variables: {
+                groupId: 'hidden-group',
+                userId: 'admin-member-user',
+              },
+            }),
+          ).resolves.toMatchObject({
+            errors: expect.arrayContaining([
+              expect.objectContaining({
+                message: 'Not Authorized!',
+              }),
+            ]),
+          })
+        })
+      })
+
+      describe('as owner', () => {
+        beforeEach(async () => {
+          authenticatedUser = await ownerMemberUser.toJson()
+        })
+
+        it('removes the user from the group', async () => {
+          await expect(
+            mutate({
+              mutation: removeUserFromGroupMutation(),
+              variables: {
+                groupId: 'hidden-group',
+                userId: 'usual-member-user',
+              },
+            }),
+          ).resolves.toMatchObject({
+            data: {
+              RemoveUserFromGroup: expect.objectContaining({
+                id: 'usual-member-user',
+                myRoleInGroup: null,
+              }),
+            },
+            errors: undefined,
+          })
+        })
+
+        it('cannot remove self', async () => {
+          await expect(
+            mutate({
+              mutation: removeUserFromGroupMutation(),
+              variables: {
+                groupId: 'hidden-group',
+                userId: 'owner-member-user',
+              },
+            }),
+          ).resolves.toMatchObject({
+            errors: expect.arrayContaining([
+              expect.objectContaining({
+                message: 'Not Authorized!',
+              }),
+            ]),
+          })
+        })
+      })
+
+      describe('as admin', () => {
```

Comment:
Like we have said we implement the admin role later. Now it behaves like a usual user …